src/subscribables/dependentObservable.js

```diff
@@ -440,9 +440,14 @@ ko.isComputed = function (instance) {
     return ko.hasPrototype(instance, ko.computed);
 };
 
+ko.isPureComputed = function (instance) {
+    return (instance && instance._state && instance._state.pure && ko.isComputed(instance)) || false;
```

Comment:
This strategy only works in debug builds, since `_state` will be a Symbol.

A better strategy may be to check if the pure functions are the same as the regular e.g. `instance.getVersion` is the same as the regular `ko.computed`/subscribable version.  (the overloaded version is here: https://github.com/zhaoyu0810/knockout/blob/master/src/subscribables/dependentObservable.js#L407)